tests/unit/test_openapi/test_parameters.py

```diff
@@ -111,7 +111,7 @@ def test_create_parameters(person_controller: Type[Controller]) -> None:
             Schema(
                 type=OpenAPIType.ARRAY,
                 items=Reference(ref="#/components/schemas/tests_unit_test_openapi_utils_Gender"),
-                examples=[[Gender.MALE]],
+                examples=[[Gender.FEMALE]],
```

Comment:
I feel like this shouldn't change?

Reply:
Okay, thanks for the thorough investigation. Maybe @adhtruong has an idea why this is happening, but right now it seems to be acceptable in this test case.